plugins/woocommerce-blocks/tests/e2e/tests/attributes-filter/attribute-filter.block_theme.spec.ts

```diff
@@ -1,24 +1,30 @@
 /**
  * External dependencies
  */
-import { test, expect } from '@woocommerce/e2e-playwright-utils';
-import { cli } from '@woocommerce/e2e-utils';
-import path from 'path';
-
-const PRODUCT_CATALOG_LINK = '/shop';
-const TEMPLATE_PATH = path.join(
-	__dirname,
-	'../shared/filters-with-product-collection.handlebars'
-);
+import {
+	test as base,
+	expect,
+	cli,
+	TemplateCompiler,
+} from '@woocommerce/e2e-utils';
 
 const blockData = {
 	name: 'Filter by Attribute',
 	slug: 'woocommerce/attribute-filter',
 	urlSearchParamWhenFilterIsApplied: 'filter_size=small&query_type_size=or',
 };
 
+const test = base.extend< { templateCompiler: TemplateCompiler } >( {
+	templateCompiler: async ( { requestUtils }, use ) => {
+		const compiler = await requestUtils.createTemplateFromFile(
+			'archive-product_filters-with-product-collection'
+		);
+		await use( compiler );
+	},
+} );
```

Comment:
I noticed that we're using this pattern in various files:

```ts
const test = base.extend< { templateCompiler: TemplateCompiler } >( {
	templateCompiler: async ( { requestUtils }, use ) => {
		const compiler = await requestUtils.createTemplateFromFile(
			'archive-product_attribute-filter'
		);
		await use( compiler );
	},
} );
```
```ts
const test = base.extend< { templateCompiler: TemplateCompiler } >( {
	templateCompiler: async ( { requestUtils }, use ) => {
		const compiler = await requestUtils.createTemplateFromFile(
			'archive-product_price-filter'
		);
		await use( compiler );
	},
} );
```
```ts
const test = base.extend< { templateCompiler: TemplateCompiler } >( {
	templateCompiler: async ( { requestUtils }, use ) => {
		const compiler = await requestUtils.createTemplateFromFile(
			'archive-product_rating-filter'
		);
		await use( compiler );
	},
} );
```

What do you think of creating a helper function for this, to prevent duplicated code in the individual test files?

Reply:
Thanks for explaining this, Bart. Let's go with the current implementation then.